frost/errors.go

```diff
@@ -91,7 +91,10 @@ func Error(code int32) error {
 }
 
 func CulpritData(culpritData []byte, code int32) *Identifier {
-	if code == int32(ErrInvalidSignatureShare) || code == int32(ErrInvalidSecretShare) || code == int32(ErrInvalidProofOfKnowledge) {
+	if code == int32(ErrInvalidSignatureShare) ||
+		code == int32(ErrInvalidSecretShare) ||
+		code == int32(ErrInvalidProofOfKnowledge) ||
+		code == int32(ErrIncorrectNumberOfCommitments) {
```

Comment:
Why we need to know which error codes produces culprit id? 
Can we make a check:
```
if len(culpritData) != 0 then copy(arr[:], culpritData)
```